cli-plugins/insight/triaging-vulnerabilities.hbs.md

```diff
@@ -118,3 +118,46 @@ The following conditions are required for this action:
 > **Note** The responsibility of assessing a vulnerability's impact is up to the person in charge of
 > triage. Images and sources with the same package and version might use the
 > package in a different way and might not have the same analysis values.
+
+## <a id='rebase-analyses'></a>Rebase multiple analyses
+
+Given the Continuous Delivery nature of TAP's Supply Chains, you might run into scenarios where you
+have performed Triage for several vulnerabilities on the latest version of your workload's image,
+and after a change is made in the source code and a new image is built and deployed, you want to
+carry forward the existing vulnerability analyses for all vulnerabilities that are still present
+in the newest version of the image. We call this process *rebase*, and you can run it with the
+following command:
+
+```console
+tanzu insight triage rebase \
+  --img-digest $TARGET-IMAGE
+  --artifact-group-uid $AG-UID
+```
+
+Where:
+
+- `IMG-DIGEST` is the digest of the image you want to rebase analyses into
+- `AG-UID` is the unique identifier for the workload that contains the image, and where existing analyses will be searched for
+
+Each time you run this command, you will be presented with the list of existing analysis that
+could be automatically rebased into your target image. The algorithm to search for this analyses
+uses the following criteria:
+
+- The analysis exists for a vulnerability that the target image is affected by and
+- Is linked to a "previous version" of an image and
+- There is no existing analysis for the same vulnerability and the target image, or their state is 'in\_triage'
+
+In this context, an image A is considered to be a previous version of an image B when they have
+the same name, different digests and image A was created before image B. This will be bounded on the workload's
+context, using the provided `--artifact-group-uid`.
+
+## Known limitations
+
+This is an experimental feature and there are some known limitations:
```

Comment:
Are we calling this an "alpha"?  I forget.  Can we swap "experimental" with "alpha" (or whatever)

Reply:
I removed this note since the whole Triage feature is still in alpha state